_pages/en/mandate.md

```diff
@@ -16,43 +16,43 @@ The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/wi
 
 ## Mission (team's reason for existence)
 
-Enable the strategic value of IT within ESDC by reducing its risks to accelerate business flexibility.
```

Comment:
I think previous wording is better. Maps directly on to War & Peace & IT, which we could add as a reference for this line

Reply:
Although the syntax is odd a bit :thinking: